### ISSUE TITLE
Safari: AJAX Submit with empty file upload field are not working

### ISSUE THREAD

**Issue by dkoether** [einpraegsam/powermail#272]
Hello,

it looks like Powermail / Parsley has a problem in Safari when submitting the form with AJAX is activated in TypoScript and the form contains an optional empty file upload field. In Chrome, Firefox & IE it works fine.

`plugin.tx_powermail.settings.setup.misc.ajaxSubmit = 1`

All necessary files are included:
- jQuery
- parsley.js
- Form.js

In Safari's Network Analyzer it looks like it is not receiving the answer (neverending cicle) and the request is missing informations:

Builded Request (not working / with file upload field):
```
Content-Type: multipart/form-data; boundary=----WebKitFormBoundaryB6h5wAr6n46CaHwe
Origin: https://www.domain.de.local
Accept: */*
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.1 Safari/605.1.15
Referer: https://www.domain.de.local/test/
X-Requested-With: XMLHttpRequest
```

Builded Request (working / without file upload field):
```
POST /test/ HTTP/1.1
Content-Type: multipart/form-data; boundary=----WebKitFormBoundary1rFqMJTlf9HigVrS
Origin: https://www.domain.de.local
Host: www.domain.de.local
Accept: */*
Connection: keep-alive
Accept-Language: de-de
Accept-Encoding: br, gzip, deflate
Cookie: _pk_id.13.612c=f5d64fe6a...6147.; _pk_ses.13.612c=*; fe_typo_user=44b5...9d6f;
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.1 Safari/605.1.15
Referer: https://www.domain.de.local/test/
Content-Length: 2032
X-Requested-With: XMLHttpRequest
```

I've tested this behaviour on multiple systems with the following versions and none of them are working:

System 1:
TYPO3 7.6
Powermail Version: 2.25.1

System 2:
TYPO3 7.6
Powermail Version 3.22.1

System 3:
TYPO3 8.7
Powermail Version: 5.6.0

When I remove the file upload field from form the AJAX submit works as wanted.

Did you notice this behaviour in the past or do you have any idea?

Thank you in advance,
Daniel

**Comment by einpraegsam** [comment]
No not yet, but sounds similar to https://github.com/einpraegsam/powermail_cond/issues/15

**Comment by einpraegsam** [comment]
My main problem on that issue is that I don't have to possibility for a testing in a Safari browser. Are you interested in debugging?

**Comment by nannione** [comment]
The upload field also breaks all powermail conditions in Safari 11.11.1 . Fields that should be hidden etc. are all visible. How can i help to debug?

**Comment by einpraegsam** [comment]
I'm going to close this issue now. This does not mean that there is no issue or there is no interest in fixing a possible problem. Anyways: Feel free to add pull requests.

**Comment by franzkugelmann** [comment]
It seems the issue is fixed with the current version 8.1.0 (and TYPO3 10.4.16)

**Comment by nannione** [comment]
I can confirm that the problem seems to be resolved with the powermail version 8.1, powermail conditions 8.0.1 and TYPO3 10.4.X .
